extensions/sql/transfer-process-store/sql_tps_proposal.md

```diff
@@ -0,0 +1,133 @@
+## Table schema
+
+```sql
+create table edc.data_request
+(
+    id                varchar                                            not null
+        constraint data_request_pk
+            primary key,
+    process_id        varchar                                            not null,
+    connector_address varchar                                            not null,
+    protocol          varchar default 'ids-multipart'::character varying not null,
+    connector_id      varchar,
+    asset_id          varchar                                            not null,
+    contract_id       varchar                                            not null,
+    data_destination  varchar                                            not null,
+    managed_resources boolean default true,
+    properties        varchar,
+    transfer_type     varchar
+);
+
+comment on column edc.data_request.data_destination is 'DataAddress serialized as JSON';
+
+comment on column edc.data_request.properties is 'java Map serialized as JSON';
+
+comment on column edc.data_request.transfer_type is 'TransferType serialized as JSON';
```

Comment:
As all three objects are pretty simple (just `Strings` in a `HashMap` or `String` / `Boolean` as field) I would do the extra effort create dedicated tables for them. This also might become handy when the EDC wants to query them or migrate the schema to another version.

<sub>Dominik Pinsel <dominik.pinsel@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md) </sub>

Reply:
similar reasoning as below